### PR TITLE
A handful of low-hanging fruit.

### PR DESCRIPTION
**Pull request by vext01**
Here's some easy next steps.

See individual commits.

### CODE REVIEW

File: src/librustc/sir.rs [hunk 1]
```diff
@@ -109,6 +109,9 @@ impl SirFuncCx {
                 let assign = self.lower_assign_stmt(place, rvalue);
                 self.push_stmt(bb, assign);
             }
+            mir::StatementKind::StorageLive(..) | mir::StatementKind::StorageDead(..) => {
+                // Ignore. Tracer doeesn't need to know about these.
```
Comment (ptersilie) [inline src/librustc/sir.rs]: Typo `doeesn't`

**Comment by ptersilie** [comment]
One typo. Rest looks good.

**Comment by vext01** [comment]
fixed.

**Comment by ptersilie** [comment]
Please squash.

**Comment by vext01** [comment]
Hold on, something odd is going on here.

**Comment by vext01** [comment]
Sorry, I messed up git. I took a branch before your last branch was merged and made a mess.

Should be fixed now. Please double check.

**Pull request merged**